task0/LinearRegression_beyzaNur.ipynb

```diff
@@ -0,0 +1,1600 @@
+{
```

Comment:
<p>ANSWERS:</p><p>1 - The score are as the following:</p><p>A. ToNthHouse=10 -&gt; 0.752</p><p>B. ToNthHouse=14 -&gt; 0.116</p><p>2 - Outliers are:</p><p>A. data(4)</p><p>B. data(4, 10, 11, 12, 13)</p><p>3 - For situation A, linear regression fits to data set; but in situation B, we might need higher degree polynomial approach.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>